.pre-commit-config.yaml

```diff
@@ -0,0 +1,38 @@
+default_stages: [commit, push]
+minimum_pre_commit_version: 2.10.1
+
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.4.0
+    hooks:
+    -   id: check-added-large-files
+    -   id: check-ast
+    -   id: check-docstring-first
+    -   id: check-merge-conflict
+    -   id: end-of-file-fixer
+    -   id: no-commit-to-branch
+        args: [--branch, master]
+    -   id: trailing-whitespace
+
+-   repo: local
```

Comment:
for run this hooks on local

Reply:
well here https://pre-commit.com/#repository-local-hooks it's recommended to use local hooks in specific cases. however let's try this approach, since I had some issues with non-local hooks. it should be fine.